Neos.Cache/Classes/Backend/RedisBackend.php

```diff
@@ -449,9 +512,15 @@ public function setCompressionLevel($compressionLevel): void
     }
 
     /**
-     * @param \Redis $redis
-     * @return void
+     * Sets the maximum number of items for batch operations
+     *
+     * @api
```

Comment:
hmm is this protected method supposed to be api?

```suggestion
```

Reply:
You are right, mistake